src/components/MultiTabs.js

```diff
@@ -1,49 +1,77 @@
 import { connect } from "react-redux";
 import PropTypes from "prop-types";
-import React from "react";
+import React, { useState } from "react";
 import styles from "./certificateViewer.scss";
 import { getTemplates, getActiveTemplateTab } from "../reducers/certificate";
 import { FeatureFlag } from "./FeatureFlag";
-import { CsvUploadButton } from "./AddressBook/CsvUploadButton";
-import Drawer from "./UI/Drawer";
+import { OverlayAddressBook, OverlayAddressBookContent } from "./UI/Overlay";
+import { useAddressBook } from "../common/hooks/useAddressBook";
 import { ButtonBordered } from "./UI/Button";
+import Drawer from "./UI/Drawer";
 
-const MultiTabs = ({ activeTab, templates, selectTemplateTab }) => (
-  <div id={styles["header-ui"]}>
-    <div className={`${styles["header-container"]} d-none d-lg-block d-xl-block`}>
-      <div id="template-tabs-list" className="nav nav-tabs">
-        {templates && templates.length > 0
-          ? templates.map((t, idx) => (
-              <li key={idx} className="nav-item">
-                <a
-                  className={`${styles.tab}
-                    ${idx === activeTab ? styles.active : ""}`}
-                  id={t.id}
+const MultiTabs = ({ activeTab, templates, selectTemplateTab }) => {
+  const [isOverlayVisible, setOverlayVisible] = useState(false);
+  const { addressBook } = useAddressBook();
+
+  return (
+    <>
+      <div id={styles["header-ui"]} className="pt-3 pt-md-0">
+        <div className={`${styles["header-container"]}`}>
+          <ul id="template-tabs-list" className="nav nav-tabs row no-gutters align-items-center">
+            <li className="nav-item col-auto col-md-auto ml-md-auto order-md-2">
+              <a href=" " className="my-auto ml-auto">
+                <ButtonBordered bg="tertiary">View another</ButtonBordered>
+              </a>
+            </li>
+            <FeatureFlag name="ADDRESS_BOOK_UPLOAD">
+              <li className="nav-item col-auto col-md-auto ml-2 order-md-2">
+                <ButtonBordered
+                  bg="tertiary"
                   onClick={() => {
-                    selectTemplateTab(idx);
+                    setOverlayVisible(!isOverlayVisible);
                   }}
-                  role="tab"
-                  aria-controls="home"
-                  aria-selected="true"
                 >
-                  {t.label}
-                </a>
+                  Upload Address Book
```

Comment:
```suggestion
                  Address Book
```

It makes sense to call it address book now that it doesnt open the dialog immediately.